pandas/core/generic.py

```diff
@@ -1892,6 +1897,29 @@ def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
             Optional specifying the datatype for columns. The SQL type should
             be a SQLAlchemy type, or a string for sqlite3 fallback connection.
 
+        Returns
+        --------
+            None
+
+        See Also
+        --------
+        pandas.io.sql.to_sql : this function will be called.
+
+        Examples
+        --------
+        >>> from sqlalchemy import create_engine
+        >>> engine = create_engine('sqlite:///example.db', echo=False)
+        >>> gen_names = lambda ids: ["User" + str(x) for x in ids]
```

Comment:
Any chance you could simplify this? It's a bit dense.

Maybe just make a simple df, display it. then show the insert. And maybe show a `pd.read_sql` to view the result.